pom.xml

```diff
@@ -16,17 +18,14 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
```

Comment:
This is [already set in the plugin parent POM](https://github.com/jenkinsci/plugin-pom/blob/master/pom.xml#L59).

```suggestion
```

Reply:
Removed in [e623ff9](https://github.com/cloudbees/groovy-cps/pull/120/commits/e623ff9584c3f46f36192c30ec5234d8e6e6215d).